src/pages/plots.jsx

```diff
@@ -15,6 +16,7 @@ const Plots = (props) => {
   const SPECIES = 0;
   const REACTIONS = 1;
   const ENVIRONMENT = 2;
+  const AEROSOLS = 3;
```

Comment:
Need these for below

```suggestion
  const AEROSOLS = 3;
  const { PART_MC = false } = JSON.parse(import.meta.env.VITE_FEATURE_FLAGS || "{}");  
```